migrations/env.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import with_statement
+
+import logging
+from logging.config import fileConfig
+
+<<<<<<< HEAD
```

Comment:
There are more merge conflcicts here. At some point did your migrations stop working? These conflicts look like they would cause the code in this file to break.